governance/votes/Executive vote - March 29, 2023.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Template - [Executive Vote] Recognized Delegate Compensation, MKR Transfers, SPF Dai Transfer, RETH-A Dust Adjustment, Monetalis Updates, PSM Parameter Normalization - March 29, 2023
+summary: TBD
+date: 2023-03-29T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Recognized Delegate Compensation, MKR Transfers, SPF Dai Transfer, RETH-A Dust Adjustment, Monetalis Updates, PSM Parameter Normalization - March 29, 2023Y
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- A total of **3,307 DAI** will be transferred to two Recognized Delegates, as detailed below.
+- A total of **307.12 MKR** will be transferred to the Collateral Engineering Services Core Unit (CES-001) and the Sustainable Ecosystem Scaling Core Unit (SES-001), as detailed below.
+- A total of **50,000 DAI** will be transferred to a designated address for the initial funding of Phoenix Labs, as detailed below.
+- The RETH-A [Debt Floor (`dust`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-floor) will be decreased, as detailed below.
+- Monetalis updates:
+	- Excess funds declaration, as detailed below.
+	- Removal of DC-IAM from RWA-007, as detailed below.
+	- Increase of the [MIP65](https://mips.makerdao.com/mips/details/MIP65) [Debt Ceiling (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling), as detailed below.
+- PSM Parameter Normalization, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
```

Comment:
```suggestion
Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **16 hours**.
```